mods/release-list-include.txt

```diff
@@ -7,7 +7,8 @@ bin/daemon.php
 bin/testargs.php
 bin/wait-for-connection
 bin/worker.php
-config/
+config/addon-sample.config.php
+config/local-sample.config.php
```

Comment:
Oh no, you excluded the samples which ARE important! 😱

the files you were looking for are 
```
config/addon.config.php
config/local.config.php
```

Reply:
It's an _include_ list as far as I understood.